backend/sim/channel/app.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2022 - See NOTICE file for copyright holders.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package channel
+
+import (
+	"math/rand"
+
+	"perun.network/go-perun/backend/sim/wallet"
+	"perun.network/go-perun/channel"
+)
+
+// AppID represents an app identifier.
+type AppID struct {
+	*wallet.Address
+}
+
+// Equal returns whether the object is equal to the given object.
```

Comment:
Isn't this doc a bit too general regarding "object" instead of AppID?

Reply:
I don't see a problem with being general here. It's still clear, no?